_data/projects/mocked-api.yml

```
@@ -0,0 +1,10 @@
+name: Mocked-API
+desc: A set of FREE Api's that allow you to fill your application with mocked data
+site: https://mocked-api.dev/
+tags:
+  - typescipt
+  - api
+  - javascript
+upforgrabs:
+  name: hacktoberfest
+  link: https://github.com/ageddesi/Mocked-API/issues?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest
```

:art: let's use the simpler URL here

```suggestion
  link: https://github.com/arungudelli/python.shiksha/labels/ds-algo-problems
```